src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js

```diff
@@ -37,7 +37,7 @@ const renderField = ({
   meta: { touched, error },
 }) => (
   <React.Fragment>
-    <input {...input} type={type} />
+    <input className={`form-input${error ? ' form-error' : ''}`} {...input} type={type} />
```

Comment:
There is a widely used package for conditional adding of the class names.
https://github.com/JedWatson/classnames
Please get familiar with it. (It's very straightforward package)